### ISSUE TITLE
Build Payments Import page

### ISSUE THREAD

**Issue by otech47** [setlife-network/trinary#352]
- Upload a file directly (CSV)
- Provide a sample CSV file that shows how column names need to be and how values should be formatted (dates and amounts)
- Checkbox for "Create clients if they don't exist" (default: Yes)

**Comment by mecoccaro** [comment]
this is related with the issue #348 ???

**Comment by otech47** [comment]
Yes this is a more specific issue and #348 is the parent Feature Epic

This is the issue that should be worked on
